summarizer/ai_applications/base/Containerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi9/python-39:1-158
-WORKDIR /locallm
-COPY requirements.txt /locallm/requirements.txt
+WORKDIR /summarizer
```

Comment:
change to `/locallm` (or update #L5, but probably better to keep them all the same for now)

Reply:
actually, I see what you did, since this isn't actually a locallm image, you made the WORKDIR more generic :)